project-keeper/src/main/java/com/exasol/projectkeeper/JavaProjectCrawlerRunner.java

```diff
@@ -50,67 +42,39 @@ public MavenProjectCrawlResult crawlProject(final Path... pomFiles) {
     }
 
     private String runCrawlerPlugin(final Path... pomFiles) {
-        final String projectList = Arrays.stream(pomFiles).map(pomFile -> pomFile.toAbsolutePath().toString()
-                // we use / instead of \ here as a fix for https://github.com/eclipse-ee4j/yasson/issues/540
-                .replace(FileSystems.getDefault().getSeparator(), "/")).collect(Collectors.joining(";"));
-        try {
-            final List<String> commandParts = new ArrayList<>(List.of(getMavenExecutable(), "--batch-mode",
-                    "-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn",
-                    "com.exasol:project-keeper-java-project-crawler:" + this.ownVersion + ":" + "crawl",
-                    "-DprojectsToCrawl=" + projectList,
-                    /*
-                     * We need to disable the model cache here since it caches the parent poms with {revision} as
-                     * version and then runs into trouble since the cache is different when reading the old pom (for
-                     * comparing dependencies).
-                     */
-                    "-Dmaven.defaultProjectBuilder.disableGlobalModelCache=true"));
-            if (this.mvnRepositoryOverride != null) {
-                commandParts.add("-Dmaven.repo.local=" + this.mvnRepositoryOverride);
-            }
-
-            LOGGER.fine(() -> "Executing command " + commandParts);
-            final Process proc = new ProcessBuilder(commandParts).redirectErrorStream(true).start();
+        final MavenProcessBuilder builder = buildMavenCommand(pomFiles);
```

Comment:
In this place I would enjoy a short explanation:
* Why does model cache no longer need to be diabled?

Reply:
In general the new implementation seems to be much more elegant.

Reply:
The code for overriding the model cache is moved to another method